### ISSUE TITLE
Use small cap for the word "CSS"

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/line-height-picker#158]
Why?
Capital letters attract too much attention in the middle of body text. 

Following Flexible Typesetting, chapter 4, 
```
export const Abbr = styled.abbr`
  font-feature-settings: "c2sc", "smcp”;
  letter-spacing: .01em;
  @supports (font-variant-caps: all-small-caps) {
    font-variant-caps: all-small-caps;
    font-feature-settings: normal;
  }
`;
```
- "c2sc" turns uppercase letters into small caps
- "smcp" turns lowercase letters into small caps
- letter-spacing is recommended to match the black-and-white balance of body text (Flexible Typesetting, chapter 4)
- `@supports` deals with the browsers that support `font-variant-caps`
- `all-small-caps` converts both uppercase and lowercase letters into small caps
- `font-feature-settings: normal` overrides its earlier declaration. Without this, `font-variant-caps` will be overridden by the `font-feature-settings` values. (see Flexible Typesetting, chapter 4)

**Comment by masakudamatsu** [comment]
We use 
```
font-feature-settings: 'smcp';
font-variant-caps: small-caps;
```
to convert only lowercase letters into small caps.

On the Preview page, the button label is "CSS code". We want to use the uppercase letter for C as the first letter in the button label.

**Comment by masakudamatsu** [comment]
This issue was solved with the commit ba7ab355ebd01001cc679ab09b9844afa87b2b6d
- We fail to add the issue number to the commit message.
